docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml

```diff
@@ -8,7 +8,7 @@
       <primary>wutool</primary>
     </indexterm></emphasis><emphasis>action </emphasis><emphasis
   role="bold">[WUID=nnn </emphasis><emphasis role="bold">] [ DALISERVER=ip ]
-  [option=value]</emphasis></para>
+  [CASSANDRASERVER=ip] [option=value]</emphasis></para>
```

Comment:
Since we no longer support Cassandra as a System Data Store, this parameter should be removed. 
We should also open a Jira to remove it from the WUTOOL usage.

Reply:
correct, this still displays on the wutools' help page. @ghalliday can you please comment

Reply:
https://track.hpccsystems.com/browse/HPCC-31177